src/main/resources/userEmailTemplate-feedbackSubmissionConfirmation.html

```diff
@@ -0,0 +1,20 @@
+Hello ${userName},
+
+<p/>
+${instructorFragment}
+
+<p/>
+The feedback responses for the following feedback session ${status}.
```

Comment:
If this template is only used for confirmation email, the placeholder `${status}` is not necessary. Just write the entire "has been submitted ..." here.